public/app/features/alerting/unified/RuleEditorRecordingRule.test.tsx

```diff
@@ -38,6 +38,8 @@ jest.mock('app/features/query/components/QueryEditorRow', () => ({
 
 jest.spyOn(config, 'getAllDataSources');
 
+jest.setTimeout(60 * 1000);
```

Comment:
Have we thought about implementing these slower tests as e2e tests? I heard people in other teams are trying out Playwright, it might be something we could consider. What are your thoughts on this?